tokio/src/runtime/scheduler/mod.rs

```diff
@@ -62,6 +69,11 @@ cfg_rt! {
     use crate::util::RngSeedGenerator;
 
     impl Handle {
+        #[track_caller]
+        pub(crate) fn current() -> Handle {
```

Comment:
Moving the `current()` function to `scheduler::Handle` makes it easier to get the conditional compilation tests to pass.

Eventually, `runtime::context` will be moved into here, and this will be the primary type to track the "current runtime." I am punting this work to a later PR.